src/components/SearchInput/SearchInput.tsx

```diff
@@ -9,11 +9,11 @@ const useStyles = makeStyles(({ palette, breakpoints }) => ({
     height: 40,
     border: `1px solid ${palette.secondary.dark}`,
     borderRadius: 10,
-    minWidth: 300,
+    // minWidth: 300,
```

Comment:
Do not leave commented out code in the source that's what source control is for

Reply:
removed commented codes